frontend/src/components/Rankings/index.tsx

```diff
@@ -434,6 +434,7 @@ function YearSelector({ years, setYear }: YearSelectorProps) {
                 <ExpandIcon />
             </MenuButton>
             <Menu>
+                <MenuItem onClick={() => setYear(null)}>All Time</MenuItem>
```

Comment:
> Because "Games All Time" is a different width than "Games 20XX", the column changes width when selecting a year

This is a good call out, and I think really small potatoes, so we could just open an issue for it and I'll knock it out later when I have a sec!